core/embed/rust/src/ui/model_mercury/layout.rs

```diff
@@ -255,14 +256,18 @@ struct ConfirmBlobParams {
     subtitle: Option<TString<'static>>,
     data: Obj,
     description: Option<TString<'static>>,
+    description_font: &'static TextStyle,
```

Comment:
Maybe implementing a new layout specifically for confirming the unknown contract would be justified here, instead of modifying the `ConfirmBlob`.

WDYT, but I guess we can do it on the layouts cleanup later.